src/content/docs/en/reference/adapter-reference.mdx

```diff
@@ -174,11 +296,30 @@ export default function createIntegration() {
 }
 ```
 
-#### Start
+The `createExports()` second argument can be useful when you need to access build-time configuration in your server entrypoint. For example, your server entrypoint might need the directory path in the build output where the resources generated by Astro are located:
+
+```js title="my-adapter/server.js"
+import { App } from 'astro/app';
+
+export function createExports(manifest, args) {
+  const assetsPath = args.assets;
+  /* ... */
+  return {};
+}
```

Comment:
I think it would be good if this showed an example of an actual exported handler function rather than an empty object

Reply:
Yeah, that's better now